src/index.html

```diff
@@ -1,13 +1,70 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="Html">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="true">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+
+  <body class="Html-Body">
+    <div
+      class="Card"
+      data-qa="card"
+    >
+      <div class="Card-Img">
+        <img
+          class="ImgOfWare"
+          src="./images/imac.jpeg"
+          alt="iMac - 27 Retina image"
+        >
+      </div>
+
+      <div class="Card-Body">
+        <div class="Card-Title">
+          <div class="Card-Description">Моноблок APPLE A1419 iMac  27" Retina 5K (MNED2UA/A)</div>
+          <p class="Card-Code">Код товара: 195434</p>
+        </div>
+
+        <div class="Card-Rating">
+          <div class="Stars">
+            <div class="Stars-Star"></div>
+            <div class="Stars-Star"></div>
+            <div class="Stars-Star"></div>
+            <div class="Stars-Star"></div>
+            <div class="Stars-Star"></div>
+          </div>
+
+          <div >
+            <p class="Reviews">Отзывов: 5</p>
+          </div>
+        </div>
+
+        <div class="Price">
+          <div class="Price-Title">Цена:</div>
+          <div class="Price-Value">69 999 грн</div>
```

Comment:
```suggestion
          <span class="Price-Title">Цена:</span>
          <span class="Price-Value">69 999 грн</span>
```
Це текстові поля. Div тут не дуже підходить